EIPS/eip-5192.md

```diff
@@ -34,6 +34,9 @@ A token with a `uint256 tokenId` may be inseparably-nound to a receiving account
 pragma solidity ^0.8.0;
 
 interface IERC5192 {
+  /// @notice Emitted when locking status is changed
+  event LockingStatusChanged(uint256 tokenId, bool status);
```

Comment:
I'm unsure if I like the readability of `LockingStatusChanged(..., bool status)`.

Naturally, what does it mean `Locking Status Changed to false`? If we reflect upon it actively, then we can probably understand that a lock is locked on `true` and open on `false.`

What about:

`event Locked(uint256 tokenId, bool state)` where if `state=true` then the token is `Locked` and if `state=false` the token is NOT `Locked`.

Reply:
How about these two events:  Locked and Unlocked

/// @notice Emitted when the locking status is changed to locked
/// @param tokenId The identifier for a token.
event Locked(uint256 tokenId);

/// @notice Emitted when the locking status is changed to unlocked
/// @param tokenId The identifier for a token.
event Unlocked(uint256 tokenId);

@TimDaub

Reply:
ok. But also we must decide what should be done if e.g. a token is minted. Must a `Unlocked(tokenId)` event be emitted? Whatever we decide on, let's please also add it to the natspec using `/// @dev`.